files/image_config/database/ping_pong_single_db_inst.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+port=$1
+until [[ $(/usr/bin/docker exec database redis-cli -p $port ping | grep -c PONG) -gt 0 ]]; do
```

Comment:
if we put inside the docker, then we do not do docker exec every time we need to ping.

Reply:
moved ping script into database docker